### PR TITLE
Backport ppc64le install iso generation

### PR DESCRIPTION
**Pull request by aaannz**
This set of commits are backports of ppc64le ISO image generation fixes from Kiwi10.

With SUMA5.0 we are based on SLEM5.5, but we also have ppc64le support so we need to have ppc64le self install iso with kiwi9 used by SLEM5.5.

This list also contains one more commit from me which actually enables grub install setup also for OFW firmwares and not only EFI firmwares.

### CODE REVIEW

**Comment by aaannz** [comment]
Tested at https://build.suse.de/project/show/home:oholecek:branches:Devel:Galaxy:Manager:5.0

**Comment by schaefi** [comment]
@aaannz can you please also fix/port the unit tests ? Thanks

**Comment by aaannz** [comment]
Sure, will do during this week.

**Comment by schaefi** [comment]
Thanks much

**Review by schaefi — APPROVED** [review]
Looks great :+1:

**Comment by aaannz** [comment]
Thanks for the review. I can't merge this PR as I don't have write access here. So as soon as you are fine with it, please merge.

**Pull request merged**
